changelog.d/5116.misc

```diff
@@ -0,0 +1 @@
+ Add time-based account expiration.
```

Comment:
this probably needs to be a `.feature` file rather than a `.misc` to get merged with the others

Reply:
Oh yes indeed.